### PR TITLE
l10n: Correct a typo and spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/gui/UserStatusSelector.qml [hunk 1]
```diff
@@ -134,7 +134,7 @@ ColumnLayout {
         TextField {
             id: userStatusMessageTextField
             Layout.fillWidth: true
-            placeholderText: qsTr("What is your Status?")
+            placeholderText: qsTr("What's your status?")
```
Comment (mgallien) [inline src/gui/UserStatusSelector.qml:137]: I would need an English native speaker to confirm but for a tooltip I am convinced that `is` form is to be used.

**Review by rakekniven — CHANGES_REQUESTED** [review]
See comments

File: src/gui/UserStatusSelector.qml (hunk 1 above)
Comment (Valdnet) [reply src/gui/UserStatusSelector.qml:137]: Well. @mgallien  is right. In English, the spelling of the questions and the official form is not abbreviated.
So we change to the version from "is" and in other applications I change.
I only leave the changed: "status".

**Review by rakekniven — APPROVED** [review]

**Review by mgallien — APPROVED** [review]

**Comment by mgallien** [comment]
Thanks

Once more, I forgot to ask you to clean history before merge.
Can you squash all commits into only one ?
I can also do it myself and merge directly if this you want.

**Comment by Valdnet** [comment]
> Can you squash all commits into only one ?

I make changes from the GitHub level and I cannot do that.

> I can also do it myself and merge directly if this you want.

@mgallien Please do this. Thanks😉.

**Pull request merged**